pom.xml

```diff
@@ -426,6 +426,32 @@
           <serverId>ossrh</serverId>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
+        <version>${version.checkstyle.plugin}</version>
```

Comment:
Where is `version.checkstyle.plugin` declared?

Reply:
in jboss-parent

Reply:
Ah it's declared in the parent pom. You don't need to declare <version> or the dependency to com.puppycrawl.tools